app/views/suggested_edit/category_index.html.erb

```diff
@@ -0,0 +1,29 @@
+<h1>Suggested Edits</h1>
+<p class="is-lead">This is a list of all (pending) suggested edits on posts in this category.</p>
```

Comment:
**suggestion** (ui): this isn't always correct; sometimes it's not pending suggested edits, but decided suggested edits

Reply:
Hm... but just "all suggested edits" wouldn't be correct either. I was thinking that the "(" and ")" indicated enough...

Reply:
You could either add a conditional and make sure it's correct based on URL parameters, or remove the "all" so that it's just a "list of suggested edits", which is never wrong.